### DISCUSSION TITLE
How can I delete options values of a product?

### DISCUSSION THREAD

**Discussion by willydavid1** [vendurehq/vendure#1688] [Q&A]
For example, I need to remove 16gb and add 32gb but I don't want 16gb to be among my options.

Can an entire option be removed? example I no longer want RAM in my product?

![image](https://user-images.githubusercontent.com/55035334/181269308-ab5f38df-0867-49d6-89d4-fa89989b5b1e.png)

**Comment by michaelbromley** [comment]
Hi,

There is a roadmap item for the next minor release which will address this:

* https://github.com/vendure-ecommerce/vendure/issues/1134
